<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgColorSettings</class>
 <widget class="QWidget" name="WdgColorSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Color Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel1_2">
       <property name="text">
        <string>Default color model for new images:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KisCmbIDList" name="cmbWorkingColorSpace">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="grpDisplay">
     <property name="title">
      <string>Display</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="chkUseSystemMonitorProfile">
        <property name="text">
         <string>Use system monitor profile</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="monitorprofileholder" native="true"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lblRenderingIntent">
          <property name="toolTip">
           <string>The icm profile for your calibrated monitor</string>
          </property>
          <property name="text">
           <string>&amp;Rendering intent:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>cmbMonitorIntent</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KComboBox" name="cmbMonitorIntent">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Perceptual</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Relative Colorimetric</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Saturation</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Absolute Colorimetric</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Add new color profile:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bnAddColorProfile">
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpProofingOptions">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Soft Proof Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="5" column="1">
       <widget class="QToolButton" name="gamutAlarm">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblProofingIntent">
        <property name="text">
         <string>Proofing Rendering Intent:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="KisColorSpaceSelector" name="proofingSpaceSelector" native="true"/>
      </item>
      <item row="5" column="2" colspan="2">
       <widget class="QCheckBox" name="ckbProofBlackPoint">
        <property name="text">
         <string>Back Point Compensation</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblGamutWarning">
        <property name="text">
         <string>Gamut Warning:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="QComboBox" name="cmbProofingIntent">
        <item>
         <property name="text">
          <string>Perceptual</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Relative Colorimetric</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Saturation</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Absolute Colorimetric</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="1" colspan="3">
       <widget class="QSlider" name="sldAdaptationState">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblAdaptationState">
        <property name="text">
         <string>Adaptation State:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpPasteBehaviour">
     <property name="title">
      <string>When Pasting Into Krita From Other Applications</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QRadioButton" name="radioPasteWeb">
        <property name="text">
         <string>Assume sRGB (like images from the web are supposed to be seen)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioPasteMonitor">
        <property name="text">
         <string>Assume monitor profile (like you see it in the other application)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioPasteAsk">
        <property name="text">
         <string>Ask each time</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="textLabel1_2_2">
        <property name="text">
         <string>Note: When copying/pasting inside Krita color info is always preserved.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="chkBlackpoint">
       <property name="text">
        <string>Use Blackpoint Compensation</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkAllowLCMSOptimization">
       <property name="text">
        <string>Allow Little CMS optimizations (uncheck when using linear light RGB or XYZ)</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisColorSpaceSelector</class>
   <extends>QWidget</extends>
   <header>widgets/kis_color_space_selector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KisCmbIDList</class>
   <extends>QComboBox</extends>
   <header>widgets/kis_cmb_idlist.h</header>
  </customwidget>
  <customwidget>
   <class>SqueezedComboBox</class>
   <extends>QComboBox</extends>
   <header>widgets/squeezedcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
